src/models.rs

```diff
@@ -286,17 +278,11 @@ impl BlockId {
     pub fn id(&self) -> &str {
         &self.0
     }
-
-    pub fn from(page_id: &PageId) -> Self {
-        BlockId(page_id.clone().0)
-    }
 }
 
-impl Identifiable for BlockId {
-    type Type = BlockId;
-
-    fn id(&self) -> &Self::Type {
-        self
+impl From<PageId> for BlockId {
+    fn from(page_id: PageId) -> Self {
+        BlockId(page_id.0.clone())
```

Comment:
I was noticing that `page_id` didn't have a `to_string()` method whereas `database_id` did. What is the `0` accessing?